cmd/helm/search.go

```diff
@@ -34,6 +34,19 @@ const searchDesc = `
 Search reads through all of the repositories configured on the system, and
 looks for matches.
 
+It will display the latest stable versions of that chart until you specify '--devel' flag 
```

Comment:
Would you mind updating the help string to look identical to the one in https://github.com/helm/helm/pull/6786? Thanks!